stdlib/Random/docs/src/index.md

```diff
@@ -26,6 +26,11 @@ unbounded integers, the interval must be specified (e.g. `rand(big.(1:6))`).
 Additionally, normal and exponential distributions are implemented for some `AbstractFloat` and
 `Complex` types, see [`randn`](@ref) and [`randexp`](@ref) for details.
 
+!!! note
+    The `MersenneTwister` used by unqualified `rand` or `randstring` calls is not suitable for 
```

Comment:
It may be not clear for everyone what is meant by "unqualified" here, what about:

> The `MersenneTwister` object used implicitly by unqualified `rand` or `randstring` calls is ...